### DISCUSSION TITLE
Counter Strike 2

### DISCUSSION THREAD

**Discussion by ddbmaster** [GameServerManagers/LinuxGSM#4330] [General]
Hallo Erstmal mein English ist Bad daher hier in Deutsch
Ich wollte CS2 Server Aufsetzen aber leider wird nur CSGO Server APP ID 740 aufgesetzt Frage wann wird das Geändert aud APP ID 730 CS2

vielen dank für ihre Rückmeldung

Mit GOOGLE Übersetzer:

Hello First of all my English is Bad so here in German
I wanted to set up CS2 server but unfortunately only CSGO server APP ID 740 is set up. Question when will this be changed to APP ID 730 CS2

Thank you for your feedback

**Comment by warmach1ne** [comment]
This is a reply on their discord server:

since we are being flooded with "Have you released CS2 server yet?" and "Why haven't you released CS2 server?" questions every few hours. I would like to give you an update.

Firstly as soon as CS2 server is available on LinuxGSM you will get an announcement. CS:GO was one of the most popular servers to host on LinuxGSM so we very much want CS2 released asap.

Secondly the reason there is no server yet is mainly that the CS2 server browser doesn't yet work in game or within steam and the appid 740 (CS:GO Server) hasn't been updated. 

We are still trying to figure out exactly what Valve have planned, but we only have speculation currently. All we have had so far is the CS2 FAQ stating community servers are "coming soon" and one changelog update mentioning dedicated servers on the game client. 

Some users here have managed to get a CS2 server running via the client, but you can only use direct connect. So, from what I can tell currently dedicated servers are not yet very stable and as per Valve are still "coming soon". Hopefully Valve will clarify and officially release the CS2 server soon. All LinuxGSM community are all keeping an eye out for news, but until that point we just have to wait and see.

On a side note we've been receiving messages requesting game servers for titles that haven't been launched yet or lack Linux compatibility. Please ensure that the game has already been released before submitting your request.

**Comment by ddbmaster** [comment]
Okay, thanks for the information
then I just have to be patient

And thank you for the reply
